ScoutSuite/providers/aws/resources/awslambda/functions.py

```diff
@@ -30,5 +29,32 @@ def _parse_function(self, raw_function):
         function_dict['version'] = raw_function.get('Version')
         function_dict['tracing_config'] = raw_function.get('TracingConfig')
         function_dict['revision_id'] = raw_function.get('RevisionId')
+
+        await self._add_role_information(function_dict, raw_function.get('Role'))
+        await self._add_access_policy_information(function_dict)
+
         return function_dict['name'], function_dict
 
+    async def _add_role_information(self, function_dict, role_id):
```

Comment:
We shouldn't be making cross-service calls in facades. If you want to merge information between services, you should do this in https://github.com/nccgroup/ScoutSuite/blob/master/ScoutSuite/providers/aws/provider.py#L54.

Reply:
As I moved the specific code to the awslambda facade, I assume is no considered cross-service call anymore?